File: JAVA PROGRAMMING/Test21.java
// 21. PRINT ODD NUMBERS FROM RANGE1 TO RANGE2
import java.util.Scanner;
public class Test21 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Range1 Value: ");
        int range1 = in.nextInt();

        System.out.println("Enter Range2 Value: ");
        int range2 = in.nextInt();

        for(int i = range1; i <= range2; i++) {
            if(i % 2 != 0) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test21.js
"use strict";
// 21. PRINT ODD NUMBERS FROM RANGE1 TO RANGE2
const func = (range1, range2) => {
    for (let i = range1; i <= range2; i++) {
        if (i % 2 != 0) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
};
func(10, 20);
func(50, 60);
func(80, 90);